src/core/tracking.js

```diff
@@ -167,6 +168,7 @@ export class Tracking {
 
   log(...args) {
     if (this._log) {
+      // eslint-disable-next-line amo/only-log-strings
```

Comment:
Why isn't `this.log` flagged by the lint rule? I think this log statement should have been flagged:

https://github.com/mozilla/addons-frontend/blob/48b2bccc4eedf79d945f1fa310a0c3ed8a7c42a6/src/core/tracking.js#L248

That one should probably change to `debug` anyway.

Reply:
The rule does not have `this` in the list of objects and not `log` in the list of methods.

Reply:
oh, I see. It's because it walks an AST.